keras/callbacks.py

```diff
@@ -539,6 +539,81 @@ def on_train_end(self, logs=None):
             print('Epoch %05d: early stopping' % (self.stopped_epoch + 1))
 
 
+class EarlyBaselineStopping(Callback):
```

Comment:
This is a specific instance of early stopping. Could we instead incorporate this functionality into `EarlyStopping` (`baseline` argument in constructor), or at least leverage shared code and make this a subclass of `EarlyStopping` (i.e. by setting `self.best =baseline`)?